src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -17,6 +18,13 @@
         return new Student[] {
             new Student(new Name("Alex Yeoh"), new Phone("87438807"), new School("NUS High School"),
                     new Year("Year 4"), new Admin())
```

Comment:
Whoops, I had changed `Year` to only allow digits, so this might fling an exception since it's expecting only "4"

Reply:
Okay, resolved

Reply:
Should Year allow more than just digits due to distinctions such as "Primary 2" and "Secondary 2" ?